plugins/visual-qontract/src/components/EscalationPolicyComponent/NextEscalationPolicyRow.tsx

```diff
@@ -60,7 +60,7 @@ export const NextEscalationPolicyRow = ({ ep }: { ep: any }) => {
           <Typography variant="button">{ep.name}</Typography>
         </Grid>
         <Grid item xs={4}>
-          {email && < ContactItem channel='email' href={"mailto:" + email} text={email} />}
+          {email && < ContactItem channel='email' href={"mailto:" + email.join(',')} text={email.join('\n')} />}
```

Comment:
I''m kind of wary of adding these fixes on the frontend - can't we send the content properly from the backend ? is this something we control  ?

Reply:
the backend is just a query to app-interface. I'm not a graphql guru so maybe there's a way to force concatenation into the result from the graphql query, but if not this is likely the best we can do.

Reply:
Could you explain what's wrong with this approach?

Reply:
I can't find anything on concatenating the list with the query.

Reply:
I didn't know what was on the backend, wasn't expecting a direct call to Graphql, if we get a raw output that we need to concatenate manually I believe then there's nothing better we could do